nailgun/entities.py

```diff
@@ -2050,6 +2050,39 @@ def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
         return {u'hostgroup': super(HostGroup, self).update_payload(fields)}
 
+    def path(self, which=None):
+        """Extend ``nailgun.entity_mixins.Entity.path``.
+        The format of the returned path depends on the value of ``which``:
+
+        clone
+            /api/hostgroups/:hostgroup_id/clone
+
+        Otherwise, call ``super``.
+
+        """
+        if which in ('clone'):
```

Comment:
This should be `if which == 'clone'` or `if which in ('clone',)` the way it is `('clone')` is returning a string iterator `['c', 'l', 'o', 'n', 'e']`

Also travis is not happy